pages/_document.js

```diff
@@ -3,12 +3,7 @@ import {Html, Head, Main, NextScript} from "next/document";
 export default function Document() {
   return (
     <Html>
-      <Head>
-        <link
-          rel="icon"
-          href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🐙</text></svg>"
-        />
-      </Head>
+      <Head></Head>
```

Comment:
You could call the `Head` component as a self closing tag. 
```suggestion
      <Head />
```

Reply:
I did not touch the <head>. It was already written incorrectly in the template.